screens/FundEmployer.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { useState } from "react";
+import { View, Text, StyleSheet } from "react-native";
+import InvestmentCard from "../components/InvestmentCard";
+import { Icon } from "@rneui/themed";
+
+export default function FundEmployer({ navigation }) {
+  const [amount, setAmount] = useState(100000);
+  const [fundName, setFundName] = useState("CushonMix");
+  const [riskLevel, setRiskLevel] = useState(
+    "CushonMix very high risk / return"
+  );
```

Comment:
Your are better off setting the fund UUID in state and then deriving the information